examples/undocumented/python/graphical/classifier_perceptron_graphical.py

```diff
@@ -1,78 +1,75 @@
-#!/usr/bin/env python
-
-import numpy as np
 import matplotlib.pyplot as plt
+import numpy as np
 import latex_plot_inits
 
-parameter_list = [[20, 5, 1., 1000, 1, None, 5], [100, 5, 1., 1000, 1, None, 10]]
+parameter_list = [[20, 5, 1, 1000, 1, None, 5], [100, 5, 1, 1000, 1, None, 10]]
+
+
+def classifier_perceptron_graphical(n=100, distance=5, learn_rate=1, max_iter=1000, num_threads=1, seed=None,
+                                    nperceptrons=5):
+    import shogun as sg
 
-def classifier_perceptron_graphical(n=100, distance=5, learn_rate=1., max_iter=1000, num_threads=1, seed=None, nperceptrons=5):
-	from shogun import RealFeatures, BinaryLabels
-	from shogun import Perceptron
-	from shogun import MSG_INFO
+    # 2D data
+    _DIM = 2
 
-	# 2D data
-	_DIM = 2
+    # To get the nice message that the perceptron has converged
+    # dummy = sg.BinaryLabels()
```

Comment:
this line is unnecessary or?
I'd also remove the comment about the info loglevel

Reply:
Yes, Perceptron now does not use SG_INFO, I will remove it.